notebook/static/notebook/js/actions.js

```diff
@@ -563,6 +563,25 @@ define(function(require){
                 env.notebook.save_widget.rename_notebook({notebook: env.notebook});
             }
         },
+        'toggle-all-cells-output-visibility':{
```

Comment:
You are aware that you contradict yourself from already existing action you have named ?